File: Java/006.java
public class Problem006 {

    public static void main(String[] args) {
        int n = 100;
        long sumOfSquares = 0;
        for (int i = 1; i <= n; i++) {
            sumOfSquares += i * i;
        }

        long sum = 0;
        for (int i = 1; i <= n; i++) {
            sum += i;
        }
        long squareOfSum = sum * sum;

        System.out.println(squareOfSum - sumOfSquares);
    }
}

File: Rust/006.rs
fn main() {
    let n: u32 = 100;
    let sum_of_squares: u32 = (1..=n).map(|x| x * x).sum();
    let sum_of_numbers: u32 = (1..=n).sum();
    let square_of_sum: u32 = sum_of_numbers * sum_of_numbers;

    println!("{}", square_of_sum - sum_of_squares);
}
